### PR TITLE
Feature/fair 526

### PR DESCRIPTION
**Pull request by ame-test**
**What changes does this PR bring?**

FAIR 526:
- this changes the cancellation email section so that if the user:
 - Has not submitted a gar, they get no cancellation email.
 - If they have submitted a GAR, they do a get a cancellation, they get submission reference.

Previously it was sending a email for all GARs, with the gar id as the reference.
- this was fine when the gar id was the reference for all users, but since then it has been replaced withthe CBP id.
- Without the GAR id, non gar emails beign cancelled,t he gar id provides no context teh user on which gar was cancelled on to the dev team for support.
- If a GAR is getting cancelled before the submission, it's cancellation does not do anything, it is the same as if the user did not create the gar int eh frist.
- - however, submitted joruneys beign cancelled do have a meaning, a journey that was happening is no happening, hence the confirmination is needed here, so the user knwo the journey is cancelled.

There are question on amend process e.g. if you cancel an amend GAR, it will cancel the previous journey, which could cause a hard to find bug where a user cancels an ammend because teh flight is ok, but has cancelled the journey, which causes issues when they go fly.

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [ ] Have you built the code locally and confirmed its working?
- [ ] Is the build confirmed to be passing on CI?
- [ ] Have you added enough relevant unit tests for your change?
- [ ] Have you added enough relevant E2E tests for your change?
- [ ] Have you updated any relevant documentation as part of this change?
- [ ] Has this change been tested against the Acceptance Criteria?
- [ ] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

**Comment by ame-test** [comment]
Tests:
image

**Comment by ame-test** [comment]
updates email, gov ntoify and example:
Screenshot 2024-06-13 at 13 17 23
Screenshot 2024-06-13 at 13 17 57

**Comment by ame-test** [comment]
airport validation being added is being it was mistakenly removed fromt he file, whcih causes an udnefined variable error.

File: src/app/garfile/view/get.controller.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ const CookieModel = require('../../../common/models/Cookie.class');
 const logger = require('../../../common/utils/logger')(__filename);
 const garApi = require('../../../common/services/garApi');
 const manifestFields = require('../../../common/seeddata/gar_manifest_fields.json');
+const airportValidation = require('../../../common/utils/airportValidation');
```
Comment (ninaknowlesHO) [inline src/app/garfile/view/get.controller.js:5]: Why did you need to add this in?

File: src/test/garfile/view/post.controller.test.js [hunk 2]
```diff
@@ -403,7 +403,7 @@ describe('GAR view post controller', () => {
     };
 
     callController().then().then(() => {
-      expect(garApiGetStub).to.have.been.calledOnceWithExactly('GAR-ID-EXAMPLE-1');
+      expect(garApiGetStub).to.have.been.calledOnceWithExactly('GAR-ID-EXAMPLE-1', true);
```
Comment (ninaknowlesHO) [inline src/test/garfile/view/post.controller.test.js:406]: What happens when it is false?

**Review by ninaknowlesHO — APPROVED** [review]

File: src/app/garfile/view/get.controller.js (hunk 1 above)
Comment (ame-test) [reply src/app/garfile/view/get.controller.js:5]: From discussion, it was that opensearch raised airport validaiton error, get controller did not have an airport validaiton reference, thus I added it.
* Seemed to be trigger on the amend get.controller.
